railties/test/application/middleware_test.rb

```diff
@@ -285,6 +285,21 @@ def index
       assert_equal "/foo/?something", env["ORIGINAL_FULLPATH"]
     end
 
+    test "Default stack includes Rack::Sendfile" do
+      boot!
+      assert_includes Rails.application.middleware, Rack::Sendfile
+    end
+
+    test "First middleware is Rack::Sendfile" do
+      boot!
+      assert_equal Rails.application.middleware.first, Rack::Sendfile
```

Comment:
I'd move these tests to middleware_stack_test. This test may break in the future and we don't need to test the content of the default middleware stack for this